packages/api/test/datasets/refresh-dataset

```diff
@@ -7,10 +7,9 @@
  */
 /* eslint-disable @typescript-eslint/no-var-requires */
 /* eslint-disable no-param-reassign */
+/* eslint-disable vitest/require-hook */
```

Comment:
what's this for?

Reply:
This file is erroneously getting picked up by our Vitest ESLint ruleset and it think that the top-level `fetch` should be in a `beforeEach()` hook.